frontend/src/app/model/form.model.ts

```diff
@@ -70,6 +73,8 @@ export interface OibCheckboxFormControl extends BaseOibFormControl {
 export interface OibScanModeFormControl extends BaseOibFormControl {
```

Comment:
I suppose this will become `BaseOibFormControl<ScanModeDTO | ProxyDTO>` once rebased on the other PR